invesalius/project.py

```diff
@@ -477,12 +474,12 @@ def export_project_to_nifti(self, filename, save_masks=True):
                 else:
                     ext = ".nii"
                     basename = filename
-                nib.save(mask_nifti, "{}_mask_{}_{}{}".format(basename, mask.index, mask.name, ext))
+                nib.save(mask_nifti, f"{basename}_mask_{mask.index}_{mask.name}{ext}")
 
 
 def Compress(folder, filename, filelist, compress=False):
```

Comment:
folder, filename are str
filelist List[str]

Reply:
I annotated `filelist` as `Dict[Union[str, os.PathLike], Union[str, os.PathLike]]` which is probably kind of confusing given the name.